zetaclient/chains/bitcoin/client/helpers.go

```diff
@@ -17,10 +19,26 @@ const (
 	// FeeRateRegnet is the hardcoded fee rate for regnet
 	FeeRateRegnet = 1
 
+	// FeeRateRegnetRBF is the hardcoded fee rate for regnet RBF
```

Comment:
Can we explain why it is hardcoded as well

Reply:
https://github.com/zeta-chain/node/pull/3396/commits/3f7bdde1620610dcd6f79b9082860d9e5ca5bf73